spec/unit/pops/types/enumeration_spec.rb

```diff
@@ -1,50 +1,51 @@
 require 'spec_helper'
 require 'puppet/pops'
 
+module Puppet::Pops::Types
```

Comment:
If rspec/mocha is leaving things behind between tests, then it's probably no worse to leave it in a module than leave in the global scope. I've never seen anything like that happen though.